.circleci/config.yml

```diff
@@ -14,647 +14,34 @@
-          fuzzer_repro: "/tmp/spark_fuzzer_repro"
-          fuzzer_name: "Spark"
-          fuzzer_exe: "_build/debug/velox/expression/tests/spark_expression_fuzzer_test"
-          fuzzer_args: " --seed ${RANDOM} --duration_sec 600 --logtostderr=1 --minloglevel=0 \ 
-          --repro_persist_path=/tmp/spark_fuzzer_repro"
-
-  linux-spark-aggregate-fuzzer-run:
-    executor: build
-    environment:
-      VELOX_DEPENDENCY_SOURCE: SYSTEM
-      simdjson_SOURCE: BUNDLED
-    steps:
-      - fuzzer-run:
-          fuzzer_output: "/tmp/spark_aggregate_fuzzer.log"
-          fuzzer_repro: "/tmp/spark_aggregate_fuzzer_repro"
-          fuzzer_name: "SparkAggregate"
-          fuzzer_exe: "_build/debug/velox/exec/tests/spark_aggregation_fuzzer_test"
-          fuzzer_args: " --seed ${RANDOM} --duration_sec 600 --logtostderr=1 --minloglevel=0 \
-          --repro_persist_path=/tmp/spark_aggregate_fuzzer_repro"
 
+workflows:
+  version: 2
 
-  linux-aggregate-fuzzer-run:
-    executor: build
-    environment:
-      VELOX_DEPENDENCY_SOURCE: SYSTEM
-      simdjson_SOURCE: BUNDLED
-    steps:
-      - fuzzer-run:
-          fuzzer_output: "/tmp/aggregate_fuzzer.log"
-          fuzzer_repro: "/tmp/aggregate_fuzzer_repro"
-          fuzzer_name: "Aggregate"
-          fuzzer_exe: "_build/debug/velox/exec/tests/velox_aggregation_fuzzer_test"
-          fuzzer_args: " --seed ${RANDOM} --duration_sec 3600 --logtostderr=1 --minloglevel=0 \ 
-          --repro_persist_path=/tmp/aggregate_fuzzer_repro"
+  path-filtering-workflow:
+    jobs:
 
-  linux-join-fuzzer-run:
-    executor: build
-    environment:
-      VELOX_DEPENDENCY_SOURCE: SYSTEM
-      simdjson_SOURCE: BUNDLED
-    steps:
-      - fuzzer-run:
-          fuzzer_output: "/tmp/join_fuzzer.log"
-          fuzzer_repro: "/tmp/join_fuzzer_repro"
-          fuzzer_name: "Join"
-          fuzzer_exe: "_build/debug/velox/exec/tests/velox_join_fuzzer_test"
-          fuzzer_args: " --seed ${RANDOM} --duration_sec 3600 --logtostderr=1 --minloglevel=0"
+      - path-filtering/filter:
+          name: check-sensitive-paths
 
-  format-check:
-    executor: check
-    steps:
-      - checkout
-      - run:
-          name: Check formatting
-          command: |
-            if ! make format-check; then
-              make format-fix
-              echo -e "\n==== Apply using:"
-              echo "patch -p1 \<<EOF"
-              git --no-pager diff
-              echo "EOF"
-              false
-            fi
+          # Format is: <regex path-to-test> <parameter-to-set> <value-of-pipeline-parameter>
+          # Regex below will filter out paths with test in them.
+          mapping: |
+            velox/expression/((?!.*test).*).*  run-longer-expression-fuzzer true
+            velox/exec/((?!.*test).*).*  run-longer-expression-fuzzer true
+            velox/common/((?!.*test).*).*  run-longer-expression-fuzzer true
+            velox/core/((?!.*test).*).* run-longer-expression-fuzzer true
```

Comment:
It would be good to also run longer for velox/vector/((?!.*test).*).*

Reply:
Sure I will add that, let me know if there are any other paths we might want to add for.